posthog/api/decide.py

```diff
@@ -268,10 +269,12 @@ def get_decide(request: HttpRequest):
 
             if feature_flags:
                 # Billing analytics for decide requests with feature flags
-                # Sample no. of decide requests with feature flags
-                if settings.DECIDE_BILLING_SAMPLING_RATE and random() < settings.DECIDE_BILLING_SAMPLING_RATE:
-                    count = int(1 / settings.DECIDE_BILLING_SAMPLING_RATE)
-                    increment_request_count(team.pk, count)
+                # Don't count if all requests are for survey targeting flags only.
+                if not all(flag.startswith(SURVEY_TARGETING_FLAG_PREFIX) for flag in feature_flags.keys()):
```

Comment:
I'm not super familiar with this code but this seems like it checks that all flags - regardless of enabled/disabled status - are surveys flags. But if someone has a flag that isn't for surveys, and that flag is disabled, then I still think we'd want to skip this. So maybe:

```suggestion
                if not all(flag.startswith(SURVEY_TARGETING_FLAG_PREFIX) for flag in active_flags.keys()):
```

Reply:
no no, feature_flags is all enabled flags.  active_flags are all enabled + not false flags.